_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+            <div class="helper-text" id="email_address_default_err_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field">
+            <label class="form-label field-required" for="email_address_default_suc_helper">Email address (default, has success)</label>
+            <input class="form-input input-text has-success" type="email" name="email_address_default_suc_helper" id="email_address_default_suc_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_suc_helper" required>
+            <div class="helper-text" id="email_address_default_suc_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field">
+            <label class="form-label" for="full_name_simple">Your name (simple text field)</label>
+            <input class="form-input input-text input-alt" type="text" name="full_name_simple" id="full_name_simple" placeholder="e.g. Jane Doe" required>
+        </div>
+        <div class="field">
+            <label class="form-label" for="full_name_simple_err">Your name (simple text field, error)</label>
+            <input class="form-input input-text input-alt has-error" type="text" name="full_name_simple_err" id="full_name_simple_err" placeholder="e.g. Jane Doe" required>
+        </div>
+        <div class="field">
+            <label class="form-label" for="full_name_simple_suc">Your name (simple text field, success)</label>
+            <input class="form-input input-text input-alt has-success" type="text" name="full_name_simple_suc" id="full_name_simple_suc" placeholder="e.g. Jane Doe" required>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Select/Dropdown/Combo</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd sr-only">Some select menus</legend>
+        <div class="field">
+            <label class="form-label" for="select_01">Default select menu</label>
+            <select class="form-input input-select" name="select_01" id="select_01">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_01_err">Default select menu with error</label>
```

Comment:
Please sync up your label naming convention - "Real label for example field (example explanation)".